src/test/java/org/fisco/bcos/channel/test/parallel/precompile/PerformanceDTTest.java

```diff
@@ -59,6 +60,10 @@
         initialize(groupId);
     }
 
+    public PerformanceDTTest() throws Exception {
+        groupId = "";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/manual/fisco/web3sdk/pullRequest?prid=4982666)